modules/blog/server/models/post.model.server.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = PostModel;
+
+//////////
+
+function PostModel (sequelize, DataTypes) {
+  var postSchema = {
+    title: {
+      allowNull: false,
+      required: true,
+      type: DataTypes.STRING
+    },
+    content: DataTypes.TEXT
+  };
+
```

Comment:
I think we mentioned a few more features like tags, author. Should these be included in the schema already?
(I don't have a preference but the answer likely depends on how likely are we to implement those features in the near future)